aether/sdk/drf/serializers.py

```diff
@@ -86,6 +86,35 @@ def get_url(self, obj, view_name, request, format):
         return result
 
 
+# https://www.django-rest-framework.org/api-guide/serializers/#dynamically-modifying-fields
+class DynamicFieldsSerializer(serializers.Serializer):
```

Comment:
Instead of repeat the same piece of code just create a Mixin and use it in both classes

```python
class DynamicFieldsSerializerMixin(object):
    # all the code comes here


class DynamicFieldsSerializer(DynamicFieldsSerializerMixin, serializers.Serializer):
    pass


class DynamicFieldsModelSerializer(DynamicFieldsSerializerMixin, serializers.ModelSerializer):
    pass
```